docs/workloads/azure/data/getting_started/generate_project.md

```diff
@@ -80,6 +80,29 @@ git push -u origin main
 
 ```
 
+If you have already created a repo in GitHub and want to push this project to the repo then follow this:
```

Comment:
Could this not be incorporated into the snippet above? I believe the above code is based on the repo already existing also?